.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,70 @@
+name: Enhancement
+description:  Suggest an improvement to an existing lotus feature.
+labels: [need/triage, kind/enhancement]
+body:
+- type: checkboxes
+  attributes:
+    label: Checklist
+    description: Please check off the following boxes before continuing to create an improvement suggestion!
+    options:
+      - label: This is **not** a new feature or an enhancement to the Filecoin protocol. If it is,  please open an [FIP issue](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0001.md).
+        required: true
+      - label: This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=hint%2Fneeds-triaging&template=feature_request.yaml) instead.
+        required: true
+      - label: This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the lotus forum](https://github.com/filecoin-project/lotus/discussions/categories/ideas) and select the category as `Ideas`.
+        required: true
+      - label: I **have** a specific, actionable, and well motivated improvement to propose.
+        required: true
+- type: dropdown 
+  id: component
+  validations:
+    required: true
+  attributes:
+    label: Lotus component
+    description: Please select the lotus component you are propoing improvement for
+    options:
+    - lotus daemon
+    - lotus miner/lotus worker
+    - lotus client
+    - lotus market
+    - lotus JSON-RPC API
+    - Other
+- type: dropdown
+  id: area
+  validations: 
+    required: true
+  attributes:
+    label: Area
+    description: Please select the area
+    options:
+    - syncing
+    - sealing
+    - mining (WinningPoSt)
+    - proving (WindowPoSt)
+    - deal making
+    - api call
+    - message management (mpool)
+    - Other
+- type: textarea
```

Comment:
I'd opt for a single free-form section, honestly. Requiring structure can help in some cases, but I usually find it just leads to redundant or pointless text.

Reply:
updated!